python3/bin/perfmon

```diff
@@ -1608,9 +1605,8 @@ if __name__ == "__main__":
         # Python built-in Exception has args,
         # but XenAPI.Failure has details instead. Sigh.
```

Comment:
This comment seems like it should be removed

Reply:
This comment explains why the below `try - except` is needed (XenAPI failure has a different format with the built-in exception).
I'll remove the first line and keep the second line so that others know why the `exception` is needed.